src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -166,14 +166,6 @@ function compute_sgs_flux!(
     massflux_h[kf_surf] = 0
     massflux_qt[kf_surf] = 0
 
-    massflux_tendency_h = aux_tc.massflux_tendency_h
-    massflux_tendency_qt = aux_tc.massflux_tendency_qt
-    # Compute the  mass flux tendencies
-    # Adjust the values of the grid mean variables
-    # Prepare the output
-    @. massflux_tendency_h = -∇c(wvec(massflux_h)) / ρ_c
-    @. massflux_tendency_qt = -∇c(wvec(massflux_qt)) / ρ_c
-
```

Comment:
We also have the `massflux_tendency_ql` and `massflux_tendency_qi` for the non-equilibrium case. Should we also delete those?